native/.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["react-native"]
+  "presets": ["module:metro-react-native-babel-preset"]
```

Comment:
As mandated here: https://github.com/react-native-community/react-native-releases/blob/master/CHANGELOG.md#updating-to-this-version